### ISSUE TITLE
postsrsd failing to start with init on 14.04

### ISSUE THREAD

**Issue by rgrue** [roehling/postsrsd#26]
Hi Timo, thanks for the great project.

I'm getting errors starting postsrsd with init. I've installed the version from your stable PPA, running this on x86_64 Ubuntu 14.04.

After running `service postsrsd start`, I get the following in my syslog:

```
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12904) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12907) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12909) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12912) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12915) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12918) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12921) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12924) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12927) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12930) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd main process ended, respawning
Dec 21 21:38:19 axon kernel: init: postsrsd main process (12933) terminated with status 1
Dec 21 21:38:19 axon kernel: init: postsrsd respawning too fast, stopped
```

I'm able to manually start postsrsd using the same command from `/etc/init/postsrsd.conf`:

```
/usr/sbin/postsrsd -4 -f"10001" -r"10002" -d"example.com" -s"/etc/postsrsd.secret" -u"nobody" -c"/var/lib/postsrsd" -X""
```

Let me know if you need any more information.

**Comment by hepabolu** [comment]
I had the same problem. Turns out the file in /etc/default/postsrsd needs to set some variables. It looks like the makefile in this repo creates a working version, but I first installed the package as set up at https://launchpad.net/~roehling

That version installed basically an empty file (just comments). Reinstalling using a clone of this repo didn't help so I had to manually change the file to make things working.
